Gruntfile.coffee

```diff
@@ -39,6 +39,7 @@ module.exports = (grunt) ->
 			"sprites"
 			"css"
 			"js"
+			"string-replace"
```

Comment:
maybe add a particular target so it's clearer what is being replaced. EX: `string-replace:MathJaxVersion`